### PR TITLE
windowsdesktop-runtime: Update to version 5.0.2

### PR DESCRIPTION
**Pull request by x-magic**
This manifest will match the latest desktop runtime version. LTS version (3.x) will be added by a separate pull request.

### CODE REVIEW

**Comment by x-magic** [comment]
Seems now fixed.

File: bucket/windowsdesktop-runtime.json [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
+            "url": "https://download.visualstudio.microsoft.com/download/pr/adeb8933-7480-4015-abf6-ca31137ad7cd/1123096ebfa5ee3f36d77500b622e4d8/windowsdesktop-runtime-5.0.2-win-x86.exe",
+            "hash": "sha512:97f6313f40159086a544f3f52cc1f34a161bd988f12c4f31b2e86eb9ffec0a72d4c9690616d80739c9f8aa65476ab1c7c4a95e804e9333c810f9d2f6d6e7ad2e"
         }
     },
     "pre_install": "if (!(is_admin)) { error 'Admin privileges are required.'; break }",
     "installer": {
         "script": "Invoke-ExternalCommand \"$dir\\$fname\" -ArgumentList '/install', '/quiet', '/norestart' -RunAs | Out-Null"
     },
+    "checkver": {
+        "url": "https://dotnetcli.azureedge.net/dotnet/Runtime/Current/latest.version",
```
Comment (Ash258) [inline bucket/windowsdesktop-runtime.json:25]:
```suggestion
        "url": "https://dotnetcli.azureedge.net/dotnet/WindowsDesktop/Current/latest.version",
```

File: bucket/windowsdesktop-runtime.json [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
-            "hash": "sha512:14a7d97fb258bee024cff3585492ed43c3ec6ac823b50980ddde8241a8bca3e578c38ca28e461630d38c180bd72323e1fcb0ee2e6e65ef9bfc8481df7beef142"
+            "url": "https://download.visualstudio.microsoft.com/download/pr/deffc9d5-ef77-4697-ac6e-33a58ccdc409/8386e478b5823a765dc1361155360877/windowsdesktop-runtime-5.0.2-win-x64.exe",
+            "hash": "sha512:6e48c1b6717040bbb431638d80cbf612c07316e9ca6bb23ee276e140d509236cdc1e8167b35148e1565a06d1f314bcdcf764597758ad3d9dbd589a6aa75d0dac"
         },
         "32bit": {
-            "url": "https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.10/windowsdesktop-runtime-3.1.10-win-x86.exe",
-            "hash": "sha512:1697e69644a1df46922e440c1ea16e583660f79f3b9b109b493a24835ae543ea6180c5167d6dbe89d8dd8991f865dba717525e333e4a2f32319733afa6a320b5"
+            "url": "https://download.visualstudio.microsoft.com/download/pr/adeb8933-7480-4015-abf6-ca31137ad7cd/1123096ebfa5ee3f36d77500b622e4d8/windowsdesktop-runtime-5.0.2-win-x86.exe",
+            "hash": "sha512:97f6313f40159086a544f3f52cc1f34a161bd988f12c4f31b2e86eb9ffec0a72d4c9690616d80739c9f8aa65476ab1c7c4a95e804e9333c810f9d2f6d6e7ad2e"
```
Comment (Ash258) [inline bucket/windowsdesktop-runtime.json:17]: fix URL

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/windowsdesktop-runtime.json (hunk 2 above)
Comment (Ash258) [reply bucket/windowsdesktop-runtime.json:17]: Why do you request review when the URLs are still random instead of the one from autoupdate??

**Comment by x-magic** [comment]
Version 5.0.3 is out. I'll open a new pull-request.

**Comment by Ash258** [comment]
No. Push it to here...

**Comment by x-magic** [comment]
> No. Push it to here...

Sorry for the trouble... I messed up my fork. Have to recreate one.

**Pull request closed without merging**